docs/integration_tests/MultipleDbContexts/README.md

```diff
@@ -0,0 +1,11 @@
+# ConfigurationOverrides class
```

Comment:
Let's structure the file differently:
1. First introduce `ConfigurationOverrides` as a general concept and what it is used for, without going into the detail on how it works (the description of `data` field is unnecessary),
2. Then describe how you can use that if you have a single `DbContext` in the app,
3. Then describe how you can use that if you have multiple `DbContext`s.